### ISSUE TITLE
arduino pro micro + 1,54'' E-Ink + Ra-01 SX1278 LoRa + NFC RFID-RC522

### ISSUE THREAD

**Issue by ESPboy-edu** [neuronspace/neuron-badge#3]
Предварительно такой комплект получается:
arduino pro micro - 200руб
WAVGAT 1,54'' E-Ink - 800руб
NFC RFID-RC522 RF - 103руб
Ra-01 SX1278 LoRa  - 200руб
CR2032  держалка - 50руб

пойдет так? прикинуть раскладку на плате?

**Comment by zhovner** [comment]
Роман, спасибо что откликнулись.

>NFC RFID-RC522 RF - 103руб

Я имел в виду только NFC антенну разведенную дорожкой на плате, полностью пассивную чтобы туда потом припаивался чип из бесконтактной карты. Футпринт чипа я найду. 

>arduino pro micro

А можем вместо ардуины вот это устройство повторить? https://www.adafruit.com/product/3406 Как я понял nRF52832 это и микроконтроллер и еще оно полностью совместимо с arduino ide

**Comment by ESPboy-edu** [comment]
можно любое ) но будет ли оно работать и как, вот в чем вопрос. я перечислил компоненты которые на 100% уверен, что заработают и не будет проблем с программированием (иде, библиотеки). и потому взял готовые законченные модули. если брать голые чипы, то там обвес нужен и в этом нужно уже хорошо понимать. я вот не понимаю в этом например. развести платы не проблема. я все еще советую на али заказать всяких компонент и попытаться спаять работающий прототип. только после этого уже плату разводить

**Comment by ESPboy-edu** [comment]
я посмотрел Feather nRF52. во первых эта плата стоит 24$ в америке и она довольно здоровая ) если вы имеете ввиду взять только чип и развести все окмпоненты, то паять smd забадаетесь и тут я не помогу. в этом случае вам нужен уже микропроцессорный инженер, который понимает все эти мелкие детальки обвеса с кварцами, кондерами, резисторами и прочим. я могу только собирать готовые штуки из цельных готовых заводских блоков-кубиков ) примерно таких, как перечислено в заголовке

**Comment by zhovner** [comment]
Согласен. Хорошо, тогда вернемся к первоначальному варианту за исключением NFC считывателя, вместо него оставляем только антенну по периметру карты. В какой программе вы разрабатываете платы?

**Comment by ESPboy-edu** [comment]
онлайновая фигня на easyeda.com
